commcare_connect/templates/tailwind/components/user_timeline.html

```diff
@@ -0,0 +1,30 @@
+<div class="bg-white shadow-sm rounded-2xl flex flex-col w-110">
```

Comment:
nit: since we are getting rid of this, no need to include it

Reply:
Yeah, will remove it.

Reply:
delete delete delete

Reply:
Done. https://github.com/dimagi/commcare-connect/pull/571/commits/07fdf99f4912c6efebf144c044196a5fb78b4063